apps/frontend/src/app/components/molecules/sample-buttons.component.ts

```diff
@@ -0,0 +1,74 @@
+import { Component } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { NbButtonModule, NbCardModule } from '@nebular/theme';
+
+@Component({
+  selector: 'tabletop-gather-sample-buttons',
+  standalone: true,
+  imports: [CommonModule, NbButtonModule, NbCardModule],
+  template: `
+    <nb-card>
+      <nb-card-header>Buttons</nb-card-header>
+      <nb-card-body class="card-body">
+        <div class="row">
+          <button *ngFor="let status of statuses" nbButton [status]="status">
+            {{ status }}
+          </button>
+        </div>
+        <div class="row">
+          <button
+            *ngFor="let status of statuses"
+            nbButton
+            outline
+            [status]="status"
+          >
+            {{ status }}
+          </button>
+        </div>
+        <div class="row">
+          <button
+            *ngFor="let status of statuses"
+            nbButton
+            hero
+            [status]="status"
+          >
+            {{ status }}
+          </button>
+        </div>
+        <div class="row">
+          <button
+            *ngFor="let status of statuses"
+            nbButton
+            ghost
+            [status]="status"
+          >
+            {{ status }}
+          </button>
+        </div>
+      </nb-card-body>
+    </nb-card>
+  `,
+  styles: [
+    `
```

Comment:
Kannst du die Styles noch im globalen Stylesheet ergänzen? Ich habe darauf geachtet keine Styles in den Components zu haben. 

Die globalen Styles sind eine Anlehnung an tailwind - ich ergänze jew. die, die es braucht :)